testing/xmrig/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Mainteiner: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=xmrig
+pkgver=2.8.3
+pkgrel=0
+pkgdesc="XMRig is a high performance Monero (XMR) miner"
+url="https://github.com/xmrig/xmrig"
+arch="x86_64 x86 aarch64 armhf armv7"
+license="GPL-3.0"
+depends="libuv libmicrohttpd openssl"
```

Comment:
This dependencies linked automatically, no need to define them
```
diff --git testing/xmrig/APKBUILD testing/xmrig/APKBUILD
index 10854f3161..371b47fcc4 100644
--- testing/xmrig/APKBUILD
+++ testing/xmrig/APKBUILD
@@ -1,5 +1,5 @@
 # Contributor: Oleg Titov <oleg.titov@gmail.com>
-# Mainteiner: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
 
 pkgname=xmrig
 pkgver=2.8.3
@@ -8,7 +8,6 @@ pkgdesc="XMRig is a high performance Monero (XMR) miner"
 url="https://github.com/xmrig/xmrig"
 arch="x86_64 x86 aarch64 armhf armv7"
 license="GPL-3.0"
-depends="libuv libmicrohttpd openssl"
 makedepends="cmake libmicrohttpd-dev libuv-dev openssl-dev"
 subpackages="$pkgname-doc"
 source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig/archive/v$pkgver.tar.gz"
```
In build log
```
>>> xmrig*: Tracing dependencies...
	so:libc.musl-x86_64.so.1
	so:libcrypto.so.1.1
	so:libmicrohttpd.so.12
	so:libssl.so.1.1
>>> xmrig*: Package size: 1004.0 KB
```

Reply:
Fixed.

Reply:
libuv dependency was not traced. Should I manually add it?